universal-login-contracts/test/contracts/Wallet/Wallet-unit.ts

```diff
@@ -1,17 +1,17 @@
 import {expect} from 'chai';
 import {deployContract, getWallets, createMockProvider} from 'ethereum-waffle';
-import {Wallet, Contract} from 'ethers';
+import {utils, Contract} from 'ethers';
 import WalletContract from '../../../build/TestableWallet.json';
-import {createKeyPair} from '@universal-login/commons';
+import {createKeyPair, TEST_GAS_PRICE, ETHER_NATIVE_TOKEN} from '@universal-login/commons';
 
 describe('UNIT: WalletContract', () => {
   let walletContract: Contract;
-  let wallet: Wallet;
 
   beforeEach(async () => {
-    [wallet] = getWallets(createMockProvider());
-    walletContract = await deployContract(wallet, WalletContract, []);
-    await walletContract.initialize(wallet.address);
+    const [, wallet] = getWallets(createMockProvider());
+    walletContract = await deployContract(wallet, WalletContract, [], {gasLimit: 5000000});
```

Comment:
extract `GAS_LIMIT_CONTTACT_DEPLOYMENT`, I think it is of by one zero should be `500000` instead of `5000000`.

Reply:
Maybe #NextPR?